timev.rb

```diff
@@ -1,112 +1,90 @@
-#
-# call-seq:
-#    Time.now -> time
-#
-# Creates a new Time object for the current time.
+# Creates a new \Time object from the current system time.
 # This is the same as Time.new without arguments.
 #
-#    Time.now            #=> 2009-06-24 12:39:54 +0900
+#    Time.now # => 2009-06-24 12:39:54 +0900
```

Comment:
Can we add an example of using the `in` keyword argument?

Reply:
Added.